src/overlays/kaleido_scope/ovl_kaleido_scope/z_kaleido_debug.c

```diff
@@ -1,25 +1,1002 @@
+                                       (INV_CONTENT(ITEM_ROOM_KEY) <= ITEM_ROOM_KEY)) {
+                                gSaveContext.save.inventory.items[slot] = INV_CONTENT(ITEM_ROOM_KEY) + 1;
+                            }
+                        } else if (CHECK_BTN_ALL(input->press.button, BTN_CLEFT)) {
+                            if (INV_CONTENT(ITEM_ROOM_KEY) == ITEM_NONE) {
+                                gSaveContext.save.inventory.items[slot] = ITEM_LETTER_MAMA;
+                            } else if ((INV_CONTENT(ITEM_ROOM_KEY) >= ITEM_LETTER_MAMA) &&
+                                       (INV_CONTENT(ITEM_ROOM_KEY) <= ITEM_LETTER_MAMA)) {
+                                gSaveContext.save.inventory.items[slot] = INV_CONTENT(ITEM_ROOM_KEY) - 1;
+                            }
+                        }
+                    } else if (CHECK_BTN_ALL(input->press.button, BTN_CRIGHT)) {
+                        if (INV_CONTENT(ITEM_LETTER_TO_KAFEI) == ITEM_NONE) {
+                            gSaveContext.save.inventory.items[slot] = ITEM_LETTER_TO_KAFEI;
+                        } else if ((INV_CONTENT(ITEM_LETTER_TO_KAFEI) >= ITEM_LETTER_TO_KAFEI) &&
+                                   (INV_CONTENT(ITEM_LETTER_TO_KAFEI) <= ITEM_LETTER_TO_KAFEI)) {
+                            gSaveContext.save.inventory.items[slot] = INV_CONTENT(ITEM_LETTER_TO_KAFEI) + 1;
+                        }
+                    } else if (CHECK_BTN_ALL(input->press.button, BTN_CLEFT)) {
+                        if (INV_CONTENT(ITEM_LETTER_TO_KAFEI) == ITEM_NONE) {
+                            gSaveContext.save.inventory.items[slot] = ITEM_PENDANT_MEMORIES;
+                        } else if ((INV_CONTENT(ITEM_LETTER_TO_KAFEI) >= ITEM_PENDANT_MEMORIES) &&
+                                   (INV_CONTENT(ITEM_LETTER_TO_KAFEI) <= ITEM_PENDANT_MEMORIES)) {
+                            gSaveContext.save.inventory.items[slot] = INV_CONTENT(ITEM_LETTER_TO_KAFEI) - 1;
+                        }
+                    }
+                } else if ((slot >= SLOT_BOTTLE_1) && (slot <= SLOT_BOTTLE_6)) {
+                    if (CHECK_BTN_ALL(input->press.button, BTN_CUP)) {
+                        value = ITEM_BOTTLE + slot - SLOT_BOTTLE_1;
+                        Inventory_DeleteItem(value, SLOT(ITEM_BOTTLE) + slot - SLOT_BOTTLE_1);
+                    } else if (CHECK_BTN_ALL(input->press.button, BTN_CRIGHT)) {
+                        if (gSaveContext.save.inventory.items[slot] == ITEM_NONE) {
+                            gSaveContext.save.inventory.items[slot] = ITEM_BOTTLE;
+                        } else if ((gSaveContext.save.inventory.items[slot] >= ITEM_BOTTLE) &&
+                                   (gSaveContext.save.inventory.items[slot] <= ITEM_HYLIAN_LOACH)) {
+                            gSaveContext.save.inventory.items[slot] = gSaveContext.save.inventory.items[slot] + 1;
```

Comment:
```suggestion
                            gSaveContext.save.inventory.items[slot]++;
```
?

Reply:
Doesn't match unfortunately, I had to go from `++` to this expanded term in my quest to match this function. Same goes for the `--` comment below